sagemaker-experiments/mnist-handwritten-digits-classification-experiment/mnist-handwritten-digits-classification-experiment.ipynb

```diff
@@ -2,7 +2,16 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    model_package_group_name = "mnist-handwritten-digit-claissification" + str(round(time.time()))
</pre><p>Typo in variable name, <code>claissification</code> -&gt; <code>classification</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
resolved in the latest push